.github/workflows/scripts/compare_translations.py

```
@@ -0,0 +1,236 @@
+"""Script to encourage more efficient coding practices.
```

Ruff (0.12.2)

The file is executable but no shebang is present

(EXE002)